org.eclipse.scanning.api/src/org/eclipse/scanning/api/malcolm/attributes/TableAttribute.java

```diff
@@ -11,6 +11,7 @@
  *******************************************************************************/
 package org.eclipse.scanning.api.malcolm.attributes;
 
+import java.util.Arrays;
 import java.util.LinkedList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.LinkedList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)